packages/atoms/booker/Booker.tsx

```diff
@@ -1,7 +1,7 @@
 import type { BookerProps } from "@calcom/features/bookings/Booker";
-import { Booker as BookerComponent } from "@calcom/features/bookings/Booker";
 
 import type { AtomsGlobalConfigProps } from "../types";
+import { BookerWebWrapper as BookerComponent } from "./wrappers/BookerWebWrapper";
```

Comment:
We cast `BookerWebWrapper` to `BookerComponent` here, and then in `BookerWebWrapper` we cast `Booker` to `BookerComponent` which I find confusing.

Shall we rename `packages/features/bookings/Booker/Booker.tsx` to `BookerUI.tsx` and then `BookerWebWrapper` to `BookerWrapper` or something?